Xamarin.Essentials/Calendars/Calendars.uwp.cs

```diff
@@ -436,22 +436,39 @@ static async Task<string> PlatformCreateCalendar(Calendar newCalendar)
         static async Task<bool> PlatformDeleteCalendarEventInstanceByDate(string eventId, string calendarId, DateTimeOffset dateOfInstanceUtc)
         {
             await Permissions.RequestAsync<Permissions.CalendarWrite>();
+            var instance = await CalendarRequest.GetInstanceAsync();
 
             if (string.IsNullOrEmpty(eventId))
             {
                 throw new ArgumentException("[UWP]: You must supply an event id to delete an event.");
             }
-            var calendarEvent = await GetEventInstanceByIdAsync(eventId, dateOfInstanceUtc);
 
-            if (calendarEvent.CalendarId != calendarId)
+            Appointment uwpAppointment;
+            try
             {
-                throw new ArgumentOutOfRangeException("[UWP]: Supplied event does not belong to supplied calendar");
+                uwpAppointment = await instance.GetAppointmentInstanceAsync(eventId, dateOfInstanceUtc);
```

Comment:
move declaration in here
ie var uwpAppointment =

Reply:
Altered accordingly